docs/reference/targets/archive.mdx

```diff
@@ -61,12 +61,21 @@ If you instead want those files included in any packages specified in the `packa
 
 <Field
     type_repr={`str | None`}
-    default_repr={`None`}
+    default_repr={`'${spec_path_normalized}/${target_name_normalized}${file_suffix}'`}
```

Comment:
I suspect this is causing the build failures, we'll need to do more escaping.

Reply:
With #235, this now looks like:
```diff
@@ -61,7 +61,7 @@ If you instead want those files included in any packages specified in the `packa
 
 <Field
     type_repr={`str | None`}
-    default_repr={`'${spec_path_normalized}/${target_name_normalized}${file_suffix}'`}
+    default_repr={`'\${spec_path_normalized}/\${target_name_normalized}\${file_suffix}'`}
 >
 
 Where the built asset should be located.
```

And renders "properly":

<img width="689" alt="image" src="https://github.com/user-attachments/assets/6a126d76-cb5a-4100-ab29-602bb8723d9a">